en/middleware/uxrce_dds.md

```diff
@@ -447,6 +455,28 @@ Each (`topic`,`type`) pairs defines:
    - `/fmu/in/` for topics that are _subscribed_ by PX4.
 4. The message type (`VehicleOdometry`, `VehicleStatus`, `OffboardControlMode`, etc.) and the ROS 2 package (`px4_msgs`) that is expected to provide the message definition.
 
+:::note
+While in general ROS2 does not allow subscribers to distinguish between multiple nodes publishing on the same topic, PX4 does implement this feature allowing uORB topics to have [multiple instances](uorb.md#multi-instance).
+For this reason, we distinguish between `subscriptions` and `subscriptions_multi`.
+:::
+
+When you add a topic to the  `subscriptions` section the uXRCE-DDS module:
+- Creates a unidirectional route going from the ROS2 topic to the _default_ instance (instance 0) of the associated uORB topic.
+For example, it creates a ROS2 subscriber of `/fmu/in/vehicle_odometry` and a uORB publisher of `vehicle_odometry`.
```

Comment:
```suggestion
  For example, it creates a ROS2 publisher of `/fmu/in/vehicle_odometry` and a uORB subscriber of `vehicle_odometry`.
```

Reply:
Hi @hamishwillee , no I did't.
The message flow goes from ROS2 to PX4, therefore uXRCE-DDS creates a subscriber on the ROS2 side and a publisher on the PX4 side.
The pipeline is
user created ROS2 node publishes topic -> uXRCE-DDS  agent subscribes to that topics -> agent/client communication -> uXRCE-DDS client advertise/publish that topic on PX4 -> PX4 modules subscribe to the topic and use the message coming from ROS2

Reply:
Duh. Sorry.